libs/internal/include/launchdarkly/network/asio_requester.hpp

```diff
@@ -68,29 +65,51 @@ static http::verb ConvertMethod(HttpMethod method) {
     launchdarkly::detail::unreachable();
 }
 
```

Comment:
Unclear to me how much this matters. This is for setting the `Host` field on the HTTP request.